app/models/exercise.rb

```diff
@@ -93,6 +93,8 @@ def download_cmd
     "exercism download --exercise=#{slug} --track=#{track.slug}".freeze
   end
 
+  def notes; end
```

Comment:
Is this left over accidentally? I can't see it being used anywhere.

Reply:
Yeah. I forgot to remove it.